components/Article/Author.js

```diff
@@ -4,17 +4,41 @@ import { css } from 'glamor'
 
 import { H1, H2, A, Interaction, Label, Lead, P } from '@project-r/styleguide'
 
+// TODO: Implement badges for different authors
 const styles = {
   name: css({
     marginRight: '10px'
+  }),
+  badge: css({
+    backgroundColor: '#ddd',
+    backgroundImage: 'url(/static/team/constantin_seibt.jpg)',
+    backgroundPosition: 'center center',
+    backgroundSize: 'cover',
+    border: '2px solid #fff',
+    borderRadius: '40px',
+    display: 'inline-block',
+    height: '80px',
+    textAlign: 'center',
+    width: '80px'
   })
 }
 
-const Author = ({ name }) => {
+const Badge = ({ show }) => {
+  if (show) {
```

Comment:
i'd avoid a `show` prop in most cases and simple ask consumers of the component to decide to render or not